scripts/sync_project.rb

```diff
@@ -74,6 +76,9 @@ def sync_firestore()
 
       # These files are integration tests, handled below
       'Firestore/Example/Tests/Integration/**',
+
+      # These files are already provided by the ProtobufCpp pod.
+      'Firestore/Protos/cpp/google/protobuf/**',
```

Comment:
This comment doesn't make sense to me and seems like it will ultimately be a source of ODR problems.

If ProtobufCpp is providing these sources why do we have them here at all?

Alternatively, if we want to keep the process of generating these sources for firestore types simple, shouldn't we exclude these from ProtobufCpp such that `Firestore/Protos/cpp` stands alone?